CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 * Make Inventory search and browse query boxes expandable. Refs UIIN-2493.
 * Added support for `containsAny` match option in Advanced search. Refs UIIN-2486.
 * Inventory search/browse: Do not retain checkbox selections when toggling search segment. Refs UIIN-2477.
+* Show Instance record after creating with Fast add option. Refs UIIN-2497.
+* Search box/Browse box- Reset all should shift focus back to search box. Refs UIIN-2514.
```

Comment:
Is the Browse box focused?

Reply:
Added a PR for Browse
https://github.com/folio-org/stripes-acq-components/pull/727